bin/jira.py

```
@@ -250,57 +283,35 @@ def get_issue_detail(self, run: str, server: bool) -> tuple:
+
+        issues = self.search_issue(run, project_name=project)
 
-        jira_data = self.search_issue(run, project_name=desk)
+        def check_issue(issue):
+            """
+            helper function to check the state of an issue
+            """
+            issue_title = issue.get("fields", {}).get("summary")
+            issue_type = issue.get("fields", {}).get("issuetype", {}).get("name")
+            return issue_type == "Sequencing Run" and issue_title[0:2] != "RE"
+        issues = list(filter(check_issues, issues))
```

Ruff (0.14.6)

Undefined name `check_issues`

(F821)